cert-expiry-finder

```diff
@@ -296,18 +296,20 @@ expiry_after() {
 
 list_k8s_cert_expiry_days() {
     local name namespace notafter unixtime delta context
-    if ! which kubectl; then
+    if ! command -v kubectl 2>/dev/null; then
         return
     fi
-    kubectl config get-contexts --no-headers -oname | while read context
+    kubectl config get-contexts --no-headers -oname | while read -r context
     do
-        kubectl get cert --context $context -A -ojson \
-            | jq -r '.items[] | (.metadata.name + " " + .metadata.namespace + " " + .status.notAfter)' \
+        kubectl get cert --context=$context -A -ojson \
+            | jq -r '.items[] | (.metadata.name + " " +
+                     .metadata.namespace + " " + .status.notAfter)' \
             | while read name namespace notafter
         do
             unixtime=$(date --date="$notafter" +%s)
             delta=$(((unixtime - global_t0) / 86400))
-            printf '%-7d Kubernetes (NS = %s, N = %s, C = %s)\n' $delta "$namespace" "$name" "$context"
+            printf '%-7d Kubernetes (NS = %s, CRT = %s, CTX = %s)\n'\
```

Comment:
Space before `\` to make it more visible.

Reply:
done